marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+                },
+                "type": "log_stream"
+            },
+            "id": 2577398194561278,
+            "layout": {
+                "height": 78,
+                "width": 47,
+                "x": 168,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "check": "marklogic.can_connect",
+                "group": "$host",
+                "group_by": [],
+                "grouping": "cluster",
+                "tags": [
+                    "*"
+                ],
+                "title": "Cluster status",
+                "title_align": "center",
+                "title_size": "16",
+                "type": "check_status"
+            },
+            "id": 8755180811202380,
+            "layout": {
+                "height": 8,
+                "width": 15,
+                "x": 24,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "check": "marklogic.forest.health",
+                "group_by": [],
+                "grouping": "cluster",
+                "tags": [
+                    "*"
+                ],
+                "title": "Forests health",
+                "title_align": "center",
+                "title_size": "16",
+                "type": "check_status"
+            },
+            "id": 2200143632431740,
+            "layout": {
+                "height": 8,
+                "width": 15,
+                "x": 40,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "check": "marklogic.database.health",
+                "group_by": [],
+                "grouping": "cluster",
+                "tags": [
+                    "*"
+                ],
+                "title": "Databases health",
+                "title_align": "center",
+                "title_size": "16",
+                "type": "check_status"
+            },
+            "id": 8505096373419124,
+            "layout": {
+                "height": 8,
+                "width": 15,
+                "x": 56,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "requests": [
+                    {
+                        "q": "top(sum:marklogic.forests.storage.disk_size{*} by {forest_name}, 10, 'last', 'desc')"
+                    }
+                ],
+                "time": {},
+                "title": "Total size of forests on disk",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "toplist"
+            },
+            "id": 7675935468546344,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 73,
+                "y": 6
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_blue",
+                "content": "Forests summary",
```

Comment:
I think the category level and naming of "Hosts," "Forests summary," and "Database summary" should be consistent. Maybe make them all categories (rather than subcategories) by making them vivid blue and naming them "# Hosts," "# Forests," and "# Databases"?